src/templates/screens.handlebars

```diff
@@ -1,45 +1,54 @@
-  <h2>{{name}}</h2>
-  {{#each topic}}
-    <span class="btn--wire topic ia-single--topic">{{this}}</span>
-  {{/each}}
-  {{#if description}}
-    <p>{{description}}</p>
-  {{/if}}
-</div>
-<a href="/edit" class="hide ia-single--edit btn--primary topic right ia-single--topic">Edit Page</a>
-<div class="ia-single--screenshots clearfix">
-  <div class="window-buttons-container">
-    <span class="circle"></span>
-    <span class="circle"></span>
-    <span class="circle"></span>
+{{#with live}}
+  <div class="left small-description hide">
+    <h2>{{name}}</h2>
+    {{#each topic}}
+      <span class="btn--wire topic ia-single--topic">{{this}}</span>
+    {{/each}}
+    {{#if description}}
+      <p>{{description}}</p>
+    {{/if}}
   </div>
-  <div class="ia-single--screenshots-top"></div>
-  <div class="ia-single--red-bar"></div>
-  <div class="ia-single--image-container">
-    <img src="https://images.duckduckgo.com/iu/?u={{encodeURIComponent 'https://ia-screenshots.s3.amazonaws.com/'}}{{id}}_index.png&f=1">
-    {{#eq dev_milestone 'live'}}
-        <div class="ia-single--serp">
-          <div class="ia-single--serp-item">
-            <div class="serp-item"></div>
-            <div class="serp-item"></div>
-            <div class="serp-item"></div>
-            <div class="serp-item"></div>
-            <div class="serp-item"></div>
-          </div>
-
-          <div class="ia-single--serp-item">
-            <div class="serp-item"></div>
-            <div class="serp-item"></div>
-            <div class="serp-item"></div>
-            <div class="serp-item"></div>
-            <div class="serp-item"></div>
-          </div>
-    {{/eq}}
-
+  <a href="/edit" class="hide ia-single--edit btn--primary topic right ia-single--topic">Edit Page</a>
+  <div class="ia-single--screenshots clearfix">
+    <div class="window-buttons-container">
+      <span class="circle"></span>
+      <span class="circle"></span>
+      <span class="circle"></span>
+    </div>
+    <div class="ia-single--screenshots-top"></div>
+    <div class="ia-single--red-bar"></div>
+    <div class="ia-single--image-container">
+      <img src="https://images.duckduckgo.com/iu/?u={{encodeURIComponent 'https://ia-screenshots.s3.amazonaws.com/'}}{{id}}_index.png&f=1">
+      {{#eq dev_milestone 'live'}}
+      <div class="ia-single--serp">
+        <div class="ia-single--serp-item">
+          <div class="serp-item"></div>
+          <div class="serp-item"></div>
+          <div class="serp-item"></div>
+          <div class="serp-item"></div>
+          <div class="serp-item"></div>
+        </div>
+        
+        <div class="ia-single--serp-item">
+          <div class="serp-item"></div>
+          <div class="serp-item"></div>
+          <div class="serp-item"></div>
+          <div class="serp-item"></div>
+          <div class="serp-item"></div>
+        </div>
+      {{/eq}}
+      
+    </div>
     </div>
   </div>
-</div>
+
+  {{#if ../permissions.can_edit}}
```

Comment:
This isn't set for the live page though. You need to modify these lines - just remove the dev milestone check and it should be fine: https://github.com/duckduckgo/community-platform/blob/jag/screenshot-button/src/ia/js/IAPage.js#L58-L64